src/animationstab.py

```
@@ -0,0 +1,1256 @@
+
+                    for name in AnimationsTab.ANIMATIONS:
+                        node_text = self._find_node_text(name, text, 1)
+                        if node_text is None:
+                            continue
+                        body = self._node_body_lines(node_text)
+                        if any(line.strip() == "off" for line in body):
+                            selections[name] = "off"
+                            continue
+                        try:
+                            overrides[name] = self._parse_node_values(name, body)
+                        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)